README.md

```diff
@@ -2,114 +2,115 @@
+to run on souped up remote infra. It's basically a flag flip.
+3. Eliminate CLI and Flask/FastAPI boilerplate by allowing you to send your function or class directly to remote
+infra to execute or serve, and keep them debuggable like the original code, not a subprocess.Popen or postman/curl call.
+4. Bake-in the middleware and automation to make your app production-quality, secure, and sharable instantly.
+That means giving you best-of-breed auth, HTTPS, telemetry, packaging, and deployment automation, with ample
+flexibility to swap in your own.
+
+## 🤨 Who is this for?
+
+* 👩‍🔧 **Engineers, Researchers and Data Scientists** who don't want to spend 3-6 months translating and packaging
+their work to share it, and want to be able to iterate and improve production services, pipelines, and artifacts
+with a Pythonic, debuggable devX.
+* 👩‍🔬 **ML and data teams** who want a versioned, shared, maintainable stack of services used across
+research and production.
+* 🦸‍♀️ **OSS maintainers** who want to supercharge their setup flow by providing a single script to stand up their app
+on any infra, rather than build support or guides for each cloud or compute system (e.g. Kubernetes) one by one.
+   * See this in action in 🤗 Hugging Face ([Transformers](https://github.com/huggingface/transformers/blob/main/examples/README.md#running-the-examples-on-remote-hardware-with-auto-setup), [Accelerate](https://github.com/huggingface/accelerate/blob/main/examples/README.md#simple-multi-gpu-hardware-launcher)) and 🦜🔗 [Langchain](https://python.langchain.com/en/latest/modules/models/llms/integrations/runhouse.html)
 
-Here is **all the code you need** to stand up a stable diffusion inference service on
-a fresh cloud GPU.
+## 🦾 How does it work?
 
+Suppose you create a cluster object:
 
 ```python
 import runhouse as rh
-from diffusers import StableDiffusionPipeline
 
-def sd_generate(prompt):
-    model = StableDiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-2-base").to("cuda")
-    return model(prompt).images[0]
-
-gpu = rh.ondemand_cluster(name="my-a100", instance_type="A100:1").up_if_not()
-sd_generate = rh.function(sd_generate).to(gpu, env=["torch", "diffusers"])
-
-# the following runs on our remote A100 gpu
-sd_generate("An oil painting of Keanu Reeves eating a sandwich.").show()
-sd_generate.save(name="sd_generate")
+gpu = rh.cluster(name="my-a100", host=my_cluster_ip, ssh_creds={"user": "ubuntu", "key": "~/.ssh/id_rsa"})
+gpu = rh.cluster(name="my-a100", instance_type="A100:1", provider="cheapest")
+gpu = rh.cluster(name="my-a10", provider="gcp", instance_type="A10:1", zone="us-west1-b", image_id="id-1332353432432", spot=True)
 ```
 
-On the data side, sync folders, tables, or blobs between local, clusters, and file storage. All
-this is done without bouncing off the laptop.
+When you send something to a cluster, we check that the cluster's up (and bring it up if not), and start Ray
```

Comment:
they don't really know what it means to "send something" to a cluster yet? 

Also -- the notion of a "cluster" confused me for a while as someone who was not coming from the Ray universe